zds/forum/managers.py

```diff
@@ -86,24 +80,19 @@ def last_topics_of_a_member(self, author, user):
         queryset = self.filter(author=author).prefetch_related("author")
         queryset = queryset.filter(self.visibility_check_query(user)).distinct()
 
-        return queryset.order_by("-pubdate").all()[: settings.ZDS_APP["forum"]["home_number"]]
+        return queryset.order_by("-pubdate").all()[:count]
 
     def get_beta_topic_of(self, tutorial):
         return self.filter(key=tutorial.pk, key__isnull=False).first()
 
-    def get_last_topics(self):
-        """
-        Get last posted topics and prefetch some related properties.
-        Depends on settings.ZDS_APP['topic']['home_number']
-        :return:
-        :rtype: django.models.Queryset
-        """
+    def get_last_topics(self, count) -> django.db.models.QuerySet:
```

Comment:
À modifier comme ceci (ou à supprimer car j'ai l'impression que c'est un ajout automatique de ton éditeur)
```suggestion
    def get_last_topics(self, count) -> models.QuerySet:
```

Reply:
Je modifie l'import, mais je le garde l'annotation.

En fait, je trouve que c'est utile de savoir que la méthode renvoie un QuerySet, parce que ce n'est pas le cas partout dans notre code. Parfois, ce sont des listes et ça change un peu la manière de s'en servir.